src/agent/at-driver.js

```diff
@@ -1,23 +1,28 @@
 import ws from 'ws';
 
 import { iterateEmitter } from '../shared/iterate-emitter.js';
+import { AgentMessage } from './messages.js';
 
 export async function createATDriver({
   url: { hostname = 'localhost', port = 4382 } = {},
   abortSignal,
+  log,
 } = {}) {
   if (!abortSignal) process.exit(1);
-  const socket = new ws(`ws://${hostname}:${port}`, ['v1.aria-at.bocoup.com']);
-  const driver = new ATDriver({ socket });
+  const socket = new ws(`ws://${hostname}:${port}/command`);
+  const driver = new ATDriver({ socket, log });
```

Comment:
I needed to pass this log param through to be able to debug log some things from inside the at-driver --- happy to back this part out, or keep it in... Thoughts?

Reply:
I committed to adding some additional logging about the at-driver communications, the additional information proved helpful again